src/timeseriesflattenerv2/_process_spec.py

```diff
@@ -152,3 +159,38 @@ def process_spec(
         ),
         pred_time_uuid_col_name=predictiontime_frame.pred_time_uuid_col_name,
     )
+
+
+def process_time_from_event_spec(
```

Comment:
Isn't currently being called by Flattener.

I think having three paths (StaticSpec, TimeDeltaSpec and TemporalSpec) is a good move for the top level function here 👍